app/views/content/non-uk-teachers/non-uk-qualifications.md

```diff
@@ -0,0 +1,135 @@
+* use the live chat service at the bottom of this page
+
+We can help you with a free digital e-statement of comparability, once you’ve submitted your application, if your provider asks for this.
+
+
+## Qualifications you will need to train to teach
+
+To train to teach in England, you will need:
+
+* a degree which meets the standard of a UK bachelor’s degree (this can come from a non-UK university)
+* an English language qualification which meets the standard of a grade 4 General Certificate of Secondary Education (GCSE)
+* a maths qualification which meets the standard of a grade 4 GCSE
+* if teaching children aged 3 to 11 in primary school, a physics, chemistry or biology (science) qualification which meets the standard of a grade 4 GCSE
+
+A GCSE is an academic qualification awarded for exams in England, usually taken at age 16.
+
+You will also need to:
+
+* have the health and physical capacity to train to teach
+* undergo safeguarding and criminal record checks carried out by your training provider or employer
+
+## English language, maths and science proficiency
+
+Your English language qualification could be evidence you’ve reached a certain standard in an English language test, for example one of the following: 
+
+* the [International English Language Testing System (IELTS)](https://www.ielts.org/)
+* [Test of English as a Foreign Language (TOEFL)](https://www.ets.org/toefl)
+* [C2 Proficiency, formerly known as Cambridge English: Proficiency (CPE)](https://www.cambridgeenglish.org/exams-and-tests/proficiency/)  
+
+If you haven’t passed an English language test like this, or do not have qualifications in maths or science which are the same standard as grade 4 GCSE, some teacher training providers will let you sit tests in these subjects. It’s best to contact them to ask about their policy before you apply. 
+
+## If you do not have a university degree
+
+If you do not have a degree, you can apply to study a 3 to 4 year [undergraduate course in England](https://www.ucas.com/postgraduate/teacher-training/applying-teacher-training/find-teacher-training-programmes) which combines a teaching degree with [qualified teacher status (QTS)](https://www.gov.uk/guidance/qualified-teacher-status-qts). When you complete your studies, you’ll be able to apply for jobs as a teacher in England.
+
+The [UK Council for International Student Affairs](https://www.ukcisa.org.uk/) has lots of helpful information about studying at an English university.
+
+## How to apply
+
+Have you spoken to one of our advisers to check have the required qualifications for teacher training in England? They can also help you submit your application. 
+
+Learn more about [how to search and apply for teacher training courses](/non-uk-teachers/train-to-teach-in-england-as-an-international-student/#search-and-apply-for-teacher-training-courses).
```

Comment:
@laurahermionetennant92 find and apply rather than search?  - think you may need to update it on the page it links to as well?